### PR TITLE
Check for errors in log after a server checkpoint

### PR DESCRIPTION
**Pull request by sebratton**
Check for errors in log after a server checkpoint. Fail test if errors found. Allow exceptions set of string.

### CODE REVIEW

**Comment by sebratton** [comment]
#build

File: dev/fattest.simplicity/src/componenttest/topology/impl/LibertyServer.java [hunk 1]
```diff
@@ -513,48 +513,53 @@ public void setCriuRestoreDisableRecovery(boolean value) {
         checkpointInfo.criuRestoreDisableRecovery = value;
     }
 
+    public LibertyServer addCheckpointRegexIgnoreMessage(String regEx) {
```
Comment (tjwatson) [inline dev/fattest.simplicity/src/componenttest/topology/impl/LibertyServer.java:516]: Why is this method not directly on `CheckpointInfo` like `setAssertNoAppRestartOnRestore`?

Comment (sebratton) [reply dev/fattest.simplicity/src/componenttest/topology/impl/LibertyServer.java:516]: No reason. We have a mix of APIs now to affect the behavior at checkpoint. Some are on the actual checkpointInfo object, most recently others have been added on the server object e.g. `public boolean isCriuRestoreDisableRecovery()`. If you'd like to move them all into the checkpoint object then I'l like to do that in a follow on fix. Perhaps using that to move checkpoint info into it's own class to cut down on the gargantuan size of LibertyServer

Comment (tjwatson) [reply dev/fattest.simplicity/src/componenttest/topology/impl/LibertyServer.java:516]: I prefer we keep the logic and method in `CheckpointInfo` for these things.  I am fine with that happening in a separate PR

**Comment by sebratton** [comment]
#Libby 
#build

**Comment by sebratton** [comment]
#Libby

**Comment by sebratton** [comment]
Build failure due to known issue unrelated to this PR.

**Review by tjwatson — APPROVED** [review]

**Comment by sebratton** [comment]
#libbybot

**Pull request merged**